common/notification/index.ts

```diff
@@ -258,19 +259,24 @@ async function deliverNotification(
     };
 
     try {
-        const channel = channels.find((it) => it.canSend(notification));
-        if (!channel) {
+        const newUser = getUserForTypeORM(user);
```

Comment:
would it be better to rename `newUser` to something else?

Reply:
Maybe newUser -> user, user -> legacyUser? ;)

Reply:
@LomyW I guess I was right with my suggestion as you can see here 😄